packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.Props.ts

```diff
@@ -31,6 +31,16 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
   onSearch?: (newValue: any) => void;
 
   /**
+   * Callback executed when the user clears the search box by either clicking 'X' or hitting escape.
+   */
+  onClearClick?: (ev?: any, onClear?: ((ev?: any) => void) | null) => void;
```

Comment:
naming, when I hit escape it's not really a click.. can we remove the `click` part?

Reply:
yeah...i'll see if I can make that work